tools/asset-sync/README.md

```diff
@@ -0,0 +1,610 @@
+
+### `Pulling a zipfile of the repository`
+
+#### Overall evaluation of `Pulling a zipfile of the repository`
+
+While it is possible to download a zipfile of a GIT repository it’s not very practical. Downloading the zip is only slightly faster than just syncing the repository but the unpacking of the zip that makes this a non-starter. For example: On my Surface Laptop 2, taking zipfile of azure-sdk-for-net and using powershell’s Expand-Archive took 8:07 to unpack. Using System.IO.Compression.ZipFile’s ExtractToDirectory took 5:06 to unpack. The decompression times alone were enough to end this investigation but even if that could be rectified the result would effectively be a read-only copy of the repository.
+
+## Exploring An External Git Repo
+
+This is where the story gets complicated. Now that recordings are no longer stored directly alongside the code that they support, the process to _initially retrieve_ and _update_ the recordings gets a bit more stilted.
+
+In a previous section, we established that another git repo is the most obvious solution.
+
+As part of this project, we have established these `assets` repos for the four main languages.
+
+- [Azure/azure-sdk-for-python-assets](https://github.com/Azure/azure-sdk-for-python-assets)
+- [Azure/azure-sdk-for-js-assets](https://github.com/Azure/azure-sdk-for-js-assets)
+- [Azure/azure-sdk-for-net-assets](https://github.com/Azure/azure-sdk-for-net-assets)
+- [Azure/azure-sdk-for-java-assets](https://github.com/Azure/azure-sdk-for-java-assets)
+
+So for a given language, we will have the following resources at play:
+
+```bash
+Azure/azure-sdk-for-<language>
+Azure/azure-sdk-for-<language>-assets
+```
+
+Given the split nature of the above, We need to talk about how the test-proxy knows **which** recordings to grab. We can't simply default to `latest main`, as that will _not_ work if we need to run tests from an earlier released version of the SDK.
+
+To get around this, we will embed a reference to an assets repo SHA into the language repository. As part of the test playback, local implementations must _retrieve_ these referenced assets at runtime. After retrieving, there will also need to be a process to sync any updated recordings _back_ to the recordings repo without an extremely large amount of manual effort.
+
+As of now, it seems the best place to locate this assets SHA is in a new file in each `sdk/<service>` directory. For most of our packages this is a safe bet. Only one team member will be updating this SHA at a time, and as such it will be easy to add onto a commit one at a time. There is no parallelization! For others, like `azure-communication` in python or `spring` in Java land, this will be complex. We will revisit these topics in the [Scenarios Walkthroughs](#scenario-walkthroughs) section.
+
+```text
+<repo-root>
+  /sdk
+    /<service>
+      recording.json
+```
+
+And within a sample recording.json file...
```

Comment:
I assume this doc still needs some updates based on renaming (i.e. assets.json) and other changes. However is this still the structure of the json file you are expecting?

Reply:
Ah I guess I didn't do an update pass on the doc. Yes it should absolutely be corrected. Thanks for the catch Wes.